### ISSUE TITLE
Is It possible use tesseractOCR in CLI/C++ project?

### ISSUE THREAD

**Issue by valentynl** [tesseract-ocr/tesseract#1336]
Hello,
### Environment

* **Tesseract Version**: 
tesseract 4.00.00dev
 leptonica-1.75.3 (Feb 20 2018, 16:23:57) [MSC v.1911 LIB Debug x64]
  libgif 5.1.4 : libjpeg 9b : libpng 1.6.34 : libtiff 4.0.9 : zlib 1.2.11 : libwebp 0.6.1 : libopenjp2 2.3.0

* **Platform**: 
Platform: Windows 7  64-bit 
Visual Studia 2017

### Current Behavior:

I tried to use tesseract in CLI/C++ project. 
When I include tesseract I got the following error:

1>MSVCRT.lib(throw_bad_alloc.obj) : error LNK2005: "public: virtual char const * __cdecl std::exception::what(void)const " (?what@exception@std@@UEBAPEBDXZ) already defined in tesseract400d.lib(tesseract400d.dll)
1>C:\Projects\BaseCRT_OCR\x64\Release\BaseCRT_OCR.exe : fatal error LNK1169: one or more multiply defined symbols found
1>Done building project "BaseCRT_OCR.vcxproj" -- FAILED.

For console project tesseract  is work fine.

Is It possible use tesseractOCR in CLI/C++ project?

Note: tesseract has been buildet via cppan accordingto wiki.

Thanks,
Regards,
Valentin

**Comment by stweil** [comment]
Yes, that should be possible. As your question is not a Tesseract issue, it is better to continue the discussion in the [user forum](https://github.com/tesseract-ocr/tesseract/wiki#support).

**Comment by valentynl** [comment]
ок, thanks, but why I can not build Tesseract  with key ''
